src/solvers/dgmulti/dg.jl

```diff
@@ -199,6 +199,9 @@ function estimate_dt(mesh::DGMultiMesh, dg::DGMulti)
   return StartUpDG.estimate_h(rd, mesh.md) / StartUpDG.inverse_trace_constant(rd)
 end
 
+dt_polydeg_scaling(dg::DGMulti) = inv(dg.basis.N+1)
+dt_polydeg_scaling(dg::DGMulti{3, <:Wedge, <: TensorProductWedge}) = inv(maximum(dg.basis.N)+1)
```

Comment:
Not strictly required right now but maybe for the future: @jlchan Do you plan to support other floating point precisions than `Float64`? If so, it may be better to just return the value to divide by and perform the division in the other functions below since the current version of `dt_polydeg_scaling` will return a `Float64`, which may pollute a `Float32`

Reply:
I'd like generality with respect to floating point types as well, but it's probably easier to take care of in a separate PR. I've filed an issue https://github.com/trixi-framework/Trixi.jl/issues/1522